### DISCUSSION TITLE
Add ability to handle long paths for zip archiver

### DISCUSSION THREAD

**Discussion by ags1234** [doublecmd/doublecmd#1581] [Suggestions]
Please add ability to handle long paths for zip archiver.

I need to zip some objects that have long names (more than 100 chars) in this long path:
`d:\abcdefghij\abcdefghi\abcdef\abcd\abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijkl\abcdefghijklmnopqr`
The target is a much shorter path, so that won't create problems.
I get this error:
![2024-04-15_100911](https://github.com/doublecmd/doublecmd/assets/21011513/8670707d-3b47-49c6-8b6c-8bbba407371d)
The result is a corrupted zip archive left on the disk.
When I try to open it I see this:
![2024-04-15_101457](https://github.com/doublecmd/doublecmd/assets/21011513/0c46a1ee-5510-433a-8037-898917338761)
Programs that I tried in exactly the same conditions that work and produce zips without errors: FreeCommander, Total Commander, Unreal Commander.

**Comment by alexx2000** [comment]
Done.

**Comment by ags1234** [comment]
Damn that was fast :)
UTF16LongName did the trick?
What build number will include this fix?

**Reply by alexx2000** [reply]
Yes. Revision 11293.

**Comment by ags1234** [comment]
Excellent, it works. Thanks!
